### PR TITLE
refactor: remove redundant imports

### PR DESCRIPTION
**Pull request by davidxia**
and change import names accordingly. My IDE was complaining about
these redundant imports.

## Checks

- [x] I've made sure the tests are passing. 
- Testing Strategy
   - [x] Unit tests
   - [ ] Manual tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: ray-operator/apis/ray/v1alpha1/zz_generated.deepcopy.go [hunk 1]
```diff
@@ -6,7 +6,7 @@
 package v1alpha1
 
 import (
-	v1 "k8s.io/api/core/v1"
```
Comment (Jeffwan) [inline ray-operator/apis/ray/v1alpha1/zz_generated.deepcopy.go:9]: em. not sure why it get changed. what's the controller-runtime version you are using?

**Review by Jeffwan — APPROVED** [review]
nice catch. It does need the import cleaning.

**Comment by Jeffwan** [comment]
@davidxia Could you help resolve rebase the master and I can help merge the PR

File: ray-operator/apis/ray/v1alpha1/zz_generated.deepcopy.go (hunk 1 above)
Comment (davidxia) [reply ray-operator/apis/ray/v1alpha1/zz_generated.deepcopy.go]: thanks, accidental find/replace, I think. Reverted

**Comment by davidxia** [comment]
rebased!

**Pull request merged**